bonsai/model.py

```diff
@@ -485,6 +485,12 @@ def _afterpass(self):
                     assert isinstance(prev.value, int)
                     codeobj._add(prev.value + 1)
 
+            elif isinstance(codeobj.value, CodeReference):
+                for prev in self.values[:i]:
+                    if isinstance(prev, CodeVariable):
+                        if prev.name == codeobj.value.name:
+                            codeobj._add(prev.value)
```

Comment:
Sorry, I just noticed a bug. There should be a break here.

Reply:
Got it! Just added the break :+1: